db/schema.rb

```diff
@@ -124,6 +124,18 @@
   add_index "users", ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
   add_index "users", ["unlock_token"], name: "index_users_on_unlock_token", unique: true, using: :btree
 
+  create_table "versions", force: :cascade do |t|
```

Comment:
I thinks versions should be item_history, I think of the item changing it's self when you say version like color change